py/desiutil/test/test_stats.py

```diff
@@ -0,0 +1,33 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""Test desiutil.setup.
+"""
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+# The line above will help with 2to3 support.
+import unittest
+import numpy as np
+import pdb
+from desiutil.stats import perc
+
+
+class TestFuncFits(unittest.TestCase):
```

Comment:
There is already a class called `TestFuncFits` and that class is masking this one.  Change the name to `TestStats`.